frontend/src/js/external-forms/form-configs/selectors.ts

```diff
@@ -63,15 +63,20 @@ export const selectFormConfigs = (
   folders: string[],
   noFoldersActive: boolean,
 ) => {
-  if ((!searchTerm || searchTerm.length === 0) && filter === "all") {
+  const noFilterSet =
+    (!searchTerm || searchTerm.length === 0) &&
+    filter === "all" &&
+    folders.length === 0 &&
+    !noFoldersActive;
```

Comment:
This is the relevant change, the rest is formatting

Reply:
Minor point: Wouldn't it make more sense to extract a common function for noFiltersSet at this point?